ocp/ansible/roles/wazi-devspaces/tasks/devspaces.yaml

```diff
@@ -0,0 +1,126 @@
+---
+# Create namespace for ibm wazi devspaces
+# tasks file for devspaces
+- include_vars: devspaces.yml
+  tags: "wazi-devspaces"
```

Comment:
Noted. 
These tags would come in handy when the playbook are manually executed on the Bootnode. one can specify the tags and choose the component they want to install - instead of using the extravars.json file. 

Please let me know if the tags can be removed. I can implement that.

Reply:
The tags have been removed as per discussions. The include_role will handle the task executions based on the extravars passed by the user.